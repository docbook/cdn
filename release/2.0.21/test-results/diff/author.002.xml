<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: author.002</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="book" class="book">
         <header deltaxml:deltaV2="A=B" class="book-titlepage">
            <h1>Unit Test: author.002</h1>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#book.2">1. Chapter Title</a>
                     </span>
                  </li>
               </ul>
            </div>
         </div>
         <article deltaxml:deltaV2="A" id="book.2" class="chapter">
            <header class="chapter-titlepage">
               <h2>Chapter 1. Chapter Title</h2>
            </header>
            <table border="0" summary="Simple list" class="simplelist">
               <tr>
                  <td>
                     <span class="member">
                        <span id="a1" class="author first-last">
                           <span class="honorific">Mr</span>. <span class="firstname">FirstFirstlast</span>
                           <span class="othername">Middlename</span>
                           <span class="surname">Surname</span>, <span class="lineage">Jr</span>
                        </span>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span id="m2" class="member">
                        <span class="author last-first">
                           <span class="surname">Surname</span>, <span class="firstname">FirstLastfirst</span>
                        </span>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <span class="author family-given">
                           <span class="surname">SURNAME</span>
                           <span class="firstname">FirstFamilygiven</span> [FAMILY Given]</span>
                     </span>
                  </td>
               </tr>
            </table>
            <p>See <a href="#a1">first author</a> or
<a href="#m2">second member</a>.</p>
         </article><div deltaxml:deltaV2="B" class="content">
            <article id="book.2" class="chapter">
               <header class="chapter-titlepage">
                  <h2>Chapter 1. Chapter Title</h2>
               </header>
               <div class="content">
                  <table border="0" summary="Simple list" class="simplelist">
                     <tr>
                        <td>
                           <span class="member">
                              <span id="a1" class="author first-last">
                                 <span class="honorific">Mr</span>. <span class="firstname">FirstFirstlast</span>
                                 <span class="othername">Middlename</span>
                                 <span class="surname">Surname</span>, <span class="lineage">Jr</span>
                              </span>
                           </span>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <span id="m2" class="member">
                              <span class="author last-first">
                                 <span class="surname">Surname</span>, <span class="firstname">FirstLastfirst</span>
                              </span>
                           </span>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <span class="member">
                              <span class="author family-given">
                                 <span class="surname">SURNAME</span>
                                 <span class="firstname">FirstFamilygiven</span> [FAMILY Given]</span>
                           </span>
                        </td>
                     </tr>
                  </table>
                  <p>See <a href="#a1">first author</a> or
<a href="#m2">second member</a>.</p>
               </div>
            </article>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>