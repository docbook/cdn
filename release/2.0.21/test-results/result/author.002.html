<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for author.002.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: author.002.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">1</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: author.002&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="book"</span> <span class="att">id="book"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="book-titlepage"</span>&gt;
            <span class="tag">&lt;h1&gt;Unit Test: author.002&lt;/h1&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#book.2"</span>&gt;1. Chapter Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff del">&lt;article <span class="att">class="chapter"</span> <span class="att">id="book.2"</span>&gt;
            <span class="tag">&lt;header <span class="att">class="chapter-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;Chapter 1. Chapter Title&lt;/h2&gt;</span>
            &lt;/header&gt;</span>
            <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">class="simplelist"</span> <span class="att">summary="Simple list"</span>&gt;
               <span class="tag">&lt;tr&gt;
                  <span class="tag">&lt;td&gt;
                     <span class="tag">&lt;span <span class="att">class="member"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="author first-last"</span> <span class="att">id="a1"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="honorific"</span>&gt;Mr&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;FirstFirstlast&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="othername"</span>&gt;Middlename&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Surname&lt;/span&gt;</span>, <span class="tag">&lt;span <span class="att">class="lineage"</span>&gt;Jr&lt;/span&gt;</span>
                        &lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/td&gt;</span>
               &lt;/tr&gt;</span>
               <span class="tag">&lt;tr&gt;
                  <span class="tag">&lt;td&gt;
                     <span class="tag">&lt;span <span class="att">class="member"</span> <span class="att">id="m2"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="author last-first"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Surname&lt;/span&gt;</span>, <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;FirstLastfirst&lt;/span&gt;</span>
                        &lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/td&gt;</span>
               &lt;/tr&gt;</span>
               <span class="tag">&lt;tr&gt;
                  <span class="tag">&lt;td&gt;
                     <span class="tag">&lt;span <span class="att">class="member"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="author family-given"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;SURNAME&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;FirstFamilygiven&lt;/span&gt;</span> [FAMILY Given]&lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/td&gt;</span>
               &lt;/tr&gt;</span>
            &lt;/table&gt;</span>
            <span class="tag">&lt;p&gt;See <span class="tag">&lt;a <span class="att">href="#a1"</span>&gt;first author&lt;/a&gt;</span> or
<span class="tag">&lt;a <span class="att">href="#m2"</span>&gt;second member&lt;/a&gt;</span>.&lt;/p&gt;</span>
         &lt;/article&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: author.002&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="book"</span> <span class="att">id="book"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="book-titlepage"</span>&gt;
            <span class="tag">&lt;h1&gt;Unit Test: author.002&lt;/h1&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#book.2"</span>&gt;1. Chapter Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;article <span class="att">class="chapter"</span> <span class="att">id="book.2"</span>&gt;
               <span class="tag">&lt;header <span class="att">class="chapter-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Chapter 1. Chapter Title&lt;/h2&gt;</span>
               &lt;/header&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">class="simplelist"</span> <span class="att">summary="Simple list"</span>&gt;
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td&gt;
                           <span class="tag">&lt;span <span class="att">class="member"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="author first-last"</span> <span class="att">id="a1"</span>&gt;
                                 <span class="tag">&lt;span <span class="att">class="honorific"</span>&gt;Mr&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;FirstFirstlast&lt;/span&gt;</span>
                                 <span class="tag">&lt;span <span class="att">class="othername"</span>&gt;Middlename&lt;/span&gt;</span>
                                 <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Surname&lt;/span&gt;</span>, <span class="tag">&lt;span <span class="att">class="lineage"</span>&gt;Jr&lt;/span&gt;</span>
                              &lt;/span&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td&gt;
                           <span class="tag">&lt;span <span class="att">class="member"</span> <span class="att">id="m2"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="author last-first"</span>&gt;
                                 <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Surname&lt;/span&gt;</span>, <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;FirstLastfirst&lt;/span&gt;</span>
                              &lt;/span&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td&gt;
                           <span class="tag">&lt;span <span class="att">class="member"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="author family-given"</span>&gt;
                                 <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;SURNAME&lt;/span&gt;</span>
                                 <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;FirstFamilygiven&lt;/span&gt;</span> [FAMILY Given]&lt;/span&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                  &lt;/table&gt;</span>
                  <span class="tag">&lt;p&gt;See <span class="tag">&lt;a <span class="att">href="#a1"</span>&gt;first author&lt;/a&gt;</span> or
<span class="tag">&lt;a <span class="att">href="#m2"</span>&gt;second member&lt;/a&gt;</span>.&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/article&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="book" class="book"><header class="book-titlepage"><h1>Unit Test: author.002</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#book.2">1. Chapter Title</a></span></li></ul></div></div><article id="book.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter Title</h2></header><table border="0" summary="Simple list" class="simplelist"><tr><td><span class="member">
<span id="a1" class="author first-last"><span class="honorific">Mr</span>. <span class="firstname">FirstFirstlast</span> <span class="othername">Middlename</span> <span class="surname">Surname</span>, <span class="lineage">Jr</span></span>
</span></td></tr><tr><td><span id="m2" class="member">
<span class="author last-first"><span class="surname">Surname</span>, <span class="firstname">FirstLastfirst</span></span>
</span></td></tr><tr><td><span class="member">
<span class="author family-given"><span class="surname">SURNAME</span> <span class="firstname">FirstFamilygiven</span> [FAMILY Given]</span>
</span></td></tr></table><p>See <a href="#a1">first author</a> or
<a href="#m2">second member</a>.</p></article></article><script src="../../resources/js/prism.js"></script></td><td><article id="book" class="book"><header class="book-titlepage"><h1>Unit Test: author.002</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#book.2">1. Chapter Title</a></span></li></ul></div></div><div class="content"><article id="book.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter Title</h2></header><div class="content"><table border="0" summary="Simple list" class="simplelist"><tr><td><span class="member">
<span id="a1" class="author first-last"><span class="honorific">Mr</span>. <span class="firstname">FirstFirstlast</span> <span class="othername">Middlename</span> <span class="surname">Surname</span>, <span class="lineage">Jr</span></span>
</span></td></tr><tr><td><span id="m2" class="member">
<span class="author last-first"><span class="surname">Surname</span>, <span class="firstname">FirstLastfirst</span></span>
</span></td></tr><tr><td><span class="member">
<span class="author family-given"><span class="surname">SURNAME</span> <span class="firstname">FirstFamilygiven</span> [FAMILY Given]</span>
</span></td></tr></table><p>See <a href="#a1">first author</a> or
<a href="#m2">second member</a>.</p></div></article></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/author.002.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>