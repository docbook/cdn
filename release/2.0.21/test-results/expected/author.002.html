<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: author.002</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="book" class="book"><header class="book-titlepage"><h1>Unit Test: author.002</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#book.2">1. Chapter Title</a></span></li></ul></div></div><article id="book.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter Title</h2></header><table border="0" summary="Simple list" class="simplelist"><tr><td><span class="member">
<span id="a1" class="author first-last"><span class="honorific">Mr</span>. <span class="firstname">FirstFirstlast</span> <span class="othername">Middlename</span> <span class="surname">Surname</span>, <span class="lineage">Jr</span></span>
</span></td></tr><tr><td><span id="m2" class="member">
<span class="author last-first"><span class="surname">Surname</span>, <span class="firstname">FirstLastfirst</span></span>
</span></td></tr><tr><td><span class="member">
<span class="author family-given"><span class="surname">SURNAME</span> <span class="firstname">FirstFamilygiven</span> [FAMILY Given]</span>
</span></td></tr></table><p>See <a href="#a1">first author</a> or
<a href="#m2">second member</a>.</p></article></article><script src="../../resources/js/prism.js"></script></body></html>